core/src/main/scala/org/http4s/jdkhttpclient/JdkHttpClient.scala

```diff
@@ -249,11 +263,17 @@ object JdkHttpClient {
 
   /** A `Client` wrapping the default `HttpClient`.
     */
+  @deprecated("Use `default` which does not return a `Resource`", "0.7.1")
   def simple[F[_]](implicit F: Async[F]): Resource[F, Client[F]] =
-    Resource.eval(defaultHttpClient[F]).flatMap(apply(_))
+    Resource.eval(default)
+
+  /** A `Client` wrapping the default `HttpClient`.
+    */
+  def default[F[_]](implicit F: Async[F]): F[Client[F]] =
```

Comment:
Was not sure what to name it so I went with `default` /shrug

Reply:
An alternative to deprecating the old `Resource`-returning functions would be to open the 0.8 release series instead, such that we don't have to invent new function names. I think I would be slightly in favor of this, as the effort for downstream users will probably be very comparable.

Reply:
I had the same feeling :) ok, let's do that. I guess you should create a series/0.8 branch I can retarget to?

Reply:
> I guess you should create a series/0.8 branch I can retarget to?

https://github.com/http4s/http4s-jdk-http-client/tree/series/0.8 :+1: